src/main/java/com/intel/genomicsdb/GenomicsDBImporter.java

```diff
@@ -45,14 +46,13 @@
 import java.util.concurrent.*;
 import java.util.stream.Collectors;
 
-import static com.googlecode.protobuf.format.JsonFormat.printToString;
 import static com.intel.genomicsdb.importer.Constants.*;
 
 /**
  * Java wrapper for vcf2tiledb - imports VCFs into TileDB/GenomicsDB.
  * All vid information is assumed to be set correctly by the user (JSON files)
  */
-public class GenomicsDBImporter {
+public class GenomicsDBImporter implements JsonFileExtensions, CallSetMapExtensions, VidMapExtensions {
```

Comment:
Not a change request, but something to ponder. Should GenomicsDBImporter be a subclass of JsonFileExtensions etc or should these simply be helper classes with static functions? I'm not sure either. For now, this is ok,

Reply:
Basically a default method is a static method for an interface that can be overridden.
This should be taken as a trait in Scala or Ruby. More info: http://ruby-naseby.blogspot.com/2008/11/traits-in-ruby.html